DataFormats/GEMRecHit/src/classes_def.xml

```diff
@@ -36,6 +36,15 @@
   <class name="edm::Wrapper<edm::RangeMap<CSCDetId,edm::OwnVector<GEMCSCSegment,edm::ClonePolicy<GEMCSCSegment> >,edm::ClonePolicy<GEMCSCSegment> > >" splitLevel="0"/>
   <class name="GEMCSCSegmentRef" splitLevel="0"/>
 
+  <class name="GEMSegment" ClassVersion="3">
+  <version ClassVersion="3" checksum="1513163288"/>
+  </class>
+  <class name="std::vector<GEMSegment*>" splitLevel="0"/>
+  <class name="edm::OwnVector<GEMSegment,edm::ClonePolicy<GEMSegment> >" splitLevel="0"/>
+  <class name="edm::RangeMap<GEMDetId,edm::OwnVector<GEMSegment,edm::ClonePolicy<GEMSegment> >,edm::ClonePolicy<GEMSegment> >" splitLevel="0"/>
```

Comment:
it's a copy-paste, used similarly for all other muon hit types, since long time ago.